js/src/lib/components/InferenceWidget/shared/WidgetQuickInput/WidgetQuickInput.svelte

```diff
@@ -16,11 +17,12 @@
 		{placeholder}
 		required={true}
 		type="text"
-		disabled={isLoading}
+		disabled={isLoading || isDisabled}
 	/>
 	<WidgetSubmitBtn
```

Comment:
Let's hide the sumbit button when disabled, no?

Reply:
same comment as here https://github.com/huggingface/hub-docs/pull/1019#discussion_r1358067690